saleor/graphql/shop/tests/test_shop.py

```diff
@@ -1334,3 +1334,55 @@ def test_get_shop_limit_info_returns_null_by_default(staff_api_client):
             }
         }
     }
+
+
+COUNTRY_FILTER_QUERY = """
+    query($filter: CountryFilterInput!) {
+        shop {
+            countries(filter: $filter){
+            code
+            }
+        }
+    }
+
+"""
```

Comment:
I will suggest adding two additional tests for empty "filter" option and for `{"attachedToShippingZones": None}`.

Reply:
empty filter option is already covered in old tests, the 'None' option I will add